purchase_batch_invoicing/tests/test_invoicing.py

```diff
@@ -66,7 +67,7 @@ def setUpClass(cls):
                     "product_qty": 1,
                     "price_unit": 100,
                     "product_uom": cls.uom1.id,
-                    "date_planned": "2016-05-12",
+                    "date_planned": date(2016, 5, 12),
```

Comment:
AFAIK, you don't need to change this

Reply:
I was wondering if it is necessary, however I have changed it because I think that if in the future the Odoo date/datetime format changes, that value will not have to be changed. What do you think ?

Reply:
No, this doesn't change any way. The python datetime object is introduced in this version, but you are always able to pass the previous string format and it's converted when needed.